README.md

```diff
@@ -1 +1,5 @@
-# cr-review-previous-project
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
- Please consider providing the student with the official [Microverse README Template
](https://github.com/microverseinc/readme-template) because it provided students with **descriptive details** on what to be included in every section of the [README](https://github.com/microverseinc/readme-template) and that will make it easier for them to use and customize.